rest/restWithParameters/api.graphql

```diff
@@ -19,17 +18,19 @@ type Root {
 }
 
 """
-Try the following sample query with and without q and v to see the effects
-Sample query:
-  query {rest(q: ["Joe Smith", "Jane Smith"] v:"New York")  { args { q v } url } }
 
+Try the following sample query for the imported schema with and without either the q or v fields requested to see the effects.
+
+curl https://ACCOUNTNAME.stepzen.net/api/FOLDERNAME/__graphql\
```

Comment:
If we assume this is the code for the endpoint being loaded, then the folder name is `miscellaneous `

Reply:
great catch!